R/uvhydrograph-data.R

```diff
@@ -226,8 +226,8 @@ getUvHydro <- function(ts, field, estimatedOnly = FALSE){
   x <- ts[[field]]$points[['time']]
   
   if(!is.null(y) & !is.null(x)){
-    
-    time <- as.POSIXct(strptime(x, "%FT%T"))
+
+    time <- fastPOSIXct(x,tz = "UTC") 
```

Comment:
Yeah, so up in the roxygen part of this file, add:
#' importsFrom fasttime fastPOSIXct
Then, make sure you redo the roxygen documentation, and then you should notice that:
importsFrom(fasttime, fastPOSIXct)
is in the NAMESPACE file